docs/standards/universal-profile/lsp0-erc725account.md

```diff
@@ -13,153 +13,235 @@ sidebar_position: 2
 
-This standard also uses the **[ERC173](https://eips.ethereum.org/EIPS/eip-173)** standard to provide ownership functions for owning and controlling the implementation contract.
+- **[ERC725Y](../lsp-background/erc725.md#erc725y---generic-data-keyvalue-store)** is a generic key-value store that enables it to **attach any information** to the smart contract even after it's been deployed.
+
+- **[ERC1271](https://eips.ethereum.org/EIPS/eip-1271)** helps to **verify the validity** of a message and signature.
+
+- **[LSP1-UniversalReceiver](../generic-standards/lsp1-universal-receiver.md)** enables **notifications about incoming or outgoing transactions** and adds custom handling and behavior based on these transactions.
+
+- **[LSP14-Ownable2Step](../generic-standards/lsp14-ownable-2-step.md)** enables a **secure ownership management** system.
+
+- **[LSP17-ContractExtension](../generic-standards/lsp17-contract-extension.md)** enables the contract to be **extended after deployment** to support new standard and functionalities.
 
 ### ERC725X - Generic Executor
 
-:::note
+:::tip
 
-See the section **["Members of address types"](https://docs.soliditylang.org/en/v0.8.11/units-and-global-variables.html?highlight=staticcall#members-of-address-types)** in the Solidity documentation for more information about the different types of calls available.
+See the **[ERC725](../lsp-background//erc725.md)** standard for more information.
+
+Check the [**execute functions**](../smart-contracts/lsp0-erc725-account.md#execute) provided by **ERC725X** that allows the contract to execute multiple operations.
+
+Check the **javascript** guides to know [**How to Send native tokens**](../../guides/universal-profile/transfer-lyx.md) or [**How to Call other contract's function**](../../guides/universal-profile/interact-with-contracts.md) using the execute function.
 
 :::
 
-This substandard enables the account to execute generic calls on any other smart contracts, including transferring native tokens along with the call. External actions are possible via a smart contract's generic [`execute(...)`](../smart-contracts/lsp0-erc725-account.md#execute) function in the smart contract. **Only the owner can perform** the operations below.
+The **ERC725X** standard enables the account to perform generic calls on other smart contracts, including transferring native tokens. External actions can be executed using the smart contract's generic `execute(...)` function and multi-calls can be done with the `execute(..)` **batch** function, but only the account owner can perform these operations.
```

Comment:
We might need to discuss for this section, as we currently have the ERC725X and ERC725Y sections as duplicate in both LSP0 and the ERC725 page.

https://docs.lukso.tech/standards/lsp-background/erc725/#erc725x

This is not ideal as:
- the content is duplicate and we need to maintain in both places (doing the same work twice)
- in LSP0, there are illustrations, while in ERC725 there are no figures.
- we could end up having one page more up to date than the other.

To be discussed with @frozeman @Hugoo @YamenMerhi  . Let me know what you think.

Reply:
agreed we keep it in both